### ISSUE TITLE
Regression: from_boomerang/minmax/ia32_elf_by_boomerang/: 'Nullable object must have a value' in Reko.ImageLoaders.Elf.Relocators.x86Relocator.RelocateEntry

### ISSUE THREAD

**Issue by rfalke** [uxmal/reko#1070]
Reko: 0.10.0.0

Subject: https://github.com/rfalke/decompiler-subjects/tree/master/from_boomerang/minmax/ia32_elf_by_boomerang

```
Error: An error occurred during decompilation.
    Nullable object must have a value.
       at System.Nullable`1.get_Value()
   at Reko.ImageLoaders.Elf.Relocators.x86Relocator.RelocateEntry(Program program, ElfSymbol sym, ElfSection referringSection, ElfRelocation rela)
   at Reko.ImageLoaders.Elf.Relocators.ElfRelocator.RelocationTable.RelocateEntries(Program program, UInt64 offStrtab, UInt64 offSymtab, UInt64 symEntrySize)
   at Reko.ImageLoaders.Elf.Relocators.ElfRelocator.RelocateDynamicSymbols(Program program)
   at Reko.ImageLoaders.Elf.Relocators.ElfRelocator32.Relocate(Program program)
   at Reko.ImageLoaders.Elf.Relocators.x86Relocator.Relocate(Program program)
   at Reko.ImageLoaders.Elf.ElfLoader.Relocate(Program program, Address addrLoad)
   at Reko.ImageLoaders.Elf.ElfImageLoader.Relocate(Program program, Address addrLoad)
   at Reko.Loading.Loader.LoadExecutable(String filename, Byte[] image, String loader, Address addrLoad)
   at Reko.Decompiler.Load(String fileName, String loaderName, Address addrLoad)
   at Reko.CmdLine.CmdLineDriver.Decompile(Dictionary`2 pArgs)
```

**Comment by rfalke** [comment]
from_boomerang/paramchain/ia32_elf_by_boomerang and from_boomerang/switch/ia32_elf_by_boomerang_cc/ also has this problem.

**Comment by uxmal** [comment]
Fixed in commit 267071f7fdb
